modules/custom/express_help/express_help.module

```diff
@@ -112,9 +112,13 @@ function _express_help_list() {
       ),
     ),
   );
+  $module_owner = module_implements('express_help');
+    foreach ($module_owner as $owner) {
+      t($owner);
+    }
   foreach ($express_help as $help) {
     foreach ($help as $key => $value) {
-      $help_list['items'][$value['short_title']] = l($value['short_title'], $value['url'], array('attributes' => array('target' => '_blank')));
+      $help_list['items'][$value['short_title']] = l($value['short_title'], $value['url'], array('attributes' => array('target' => '_blank'))) . "<br><small>" . $owner . '</small>';
```

Comment:
I think you will have to add something to all of the help hooks, since they return values unsorted and don't include where they cam from. You could use something like:
```
  return array_map(
    function($a) {
      $a[0]['module'] = 'name_of_bundle';
      return $a;
    },
    $variables);
```
instead of just returning the variables. That will put the module name in the $exrpess_help array and you can print it out in this line.